packages/react-components/react-toast/library/src/components/ToastContainer/renderToastContainer.tsx

```diff
@@ -17,12 +17,12 @@ export const renderToastContainer_unstable = (
 
   return (
     <ToastContainerContextProvider value={contextValues.toast}>
-      <ToastContainerMotion appear onMotionFinish={onMotionFinish} visible={visible} unmountOnExit>
+      <CollapseDelayed appear onMotionFinish={onMotionFinish} visible={visible} unmountOnExit>
```

Comment:
I noticed the addition of the `CollapseDelayed` component directly in the `ToastContainerContextProvider`. In the past two PRs, we updated components like `Accordion` and `Tree` by defining motion elements as slots (e.g., `collapseMotion`). Should we consider following the same pattern here by defining `CollapseDelayed` as a slot?

Reply:
In the cases of `Accordion` and `Tree`, we were adding new motion, whereas this is migrating `Toast` motion that already existed.